challenge.js

```diff
@@ -2,12 +2,12 @@
 
 /** QUESTION 1
  *
- *	In plain English, how would you add a new HTML element
- *		to a web page without refreshing it?
+ *	In plain English, how would you add a new HTML element to a web page without refreshing it?
  *
  *	Answer below.
  */
-
+ we would find the elementid by rightlcikcing on an element and finding its Id in the console, you would then set a variable to a document.getElementById('id string')
+ you would hit enter and then use . notation, starting with that variable to reference something like var.style.background-color = 'green', to name one example.
```

Comment:
Great, you retrieved the element and stored the reference in a variable. You can now manipulate the DOM immediately surrounding that element through it in various ways. 1